src/pip/_internal/utils/unstable.py

```diff
@@ -0,0 +1,26 @@
+
+from pip._internal.exceptions import UnknownUnstableFeatures
+
+
+class UnstableFeaturesHelper(object):
+    """Handles logic for registering/validating/checking
+    """
+
+    def __init__(self):
+        super(UnstableFeaturesHelper, self).__init__()
```

Comment:
I don't think it's needed to call `super()` here. What does it do?

Reply:
It's out of a habit of writing cooperative classes from the get go.

From https://rhettinger.wordpress.com/2011/05/26/super-considered-super/:

> super() is in the business of delegating method calls to some class in the instance’s ancestor tree. For reorderable method calls to work, the classes need to be designed cooperatively. This presents three easily solved practical issues:
>
>  - the method being called by super() needs to exist
>  - the caller and callee need to have a matching argument signature
>  - and every occurrence of the method needs to use super()

This is in line with the 3rd point. I agree the second point makes this kind of moot but hey, it doesn't hurt us in any way so I'd rather just let this sit. :P